app/code/community/RicardoMartins/PagBank/etc/system.xml

```diff
@@ -43,6 +43,27 @@
                             <comment><![CDATA[Alguns eventos precisam ser disparados no momento da finalização da compra. Se você usa um checkout customizado, considere adicionar o identificador do seu botão de checkout. Deixe em branco se não souber. <a href="//pagsegurotransparente.zendesk.com/hc/pt-br/articles/360039820711" target="_blank">Saiba mais.</a>]]></comment>
                             <tooltip><![CDATA[Exemplos: <br/>por classe css: .btn-checkout <br/>por id: #btn-checkout. Você pode informar mais de um, separados por vírgula.]]></tooltip>
                         </placeorder_button>
+                        <hash_email_active>
+                            <label>Use hash@domain instead of the buyer's email</label>
+                            <frontend_type>select</frontend_type>
+                            <source_model>adminhtml/system_config_source_yesno</source_model>
+                            <sort_order>35</sort_order>
+                            <show_in_default>1</show_in_default>
+                            <show_in_website>1</show_in_website>
+                            <show_in_store>0</show_in_store>
+                            <tooltip>Using this feature can increase the number of denied orders and also prevents PagSeguro from sending emails about the order to the customer.</tooltip>
+                            <comment>Converts the buyer's email to hash@domain.</comment>
+                        </hash_email_active>
+                        <hide_order_items>
+                            <label>Hide order items</label>
+                            <frontend_type>select</frontend_type>
+                            <source_model>adminhtml/system_config_source_yesno</source_model>
+                            <sort_order>36</sort_order>
+                            <show_in_default>1</show_in_default>
+                            <show_in_website>1</show_in_website>
+                            <show_in_store>0</show_in_store>
+                            <comment>Hide order items in the payment request sent to PagBank.</comment>
```

Comment:
Hide order items in the payment request sent to PagBank (required in credit card transactions)